.github/workflows/openapi-sdk-compatibility-check.yml

```diff
@@ -1,18 +1,9 @@
-name: "Smart Events UI :: SDK <-> Openapi compatibility check"
+name: "Smart Events :: SDK :: Compatibility check :: (MGDOBR-868)"
 on: [pull_request, push]
```

Comment:
Is it useful to keep it enabled until MGDOBR-868 is fixed?

Reply:
@danielezonca thank you for your comment. I removed the old compatibility check. It needs to be rewritten as part of MGDOBR-868.